perun/postprocess/moving_average/methods.py

```diff
@@ -0,0 +1,221 @@
+"""
+Module with computational method of moving average and auxiliary methods at executing of this method.
+"""
+import click
+import numpy as np
+import pandas as pd
+import sklearn.metrics
+
+import perun.postprocess.regression_analysis.tools as tools
+
+# required precision for moving average models when the window width was not entered
+_MIN_R_SQUARE = .88
+# increase of the window width value in each loop, if it has not been reached the required precision
+_WINDOW_WIDTH_INCREASE = .15
+# starting window width as the part of the length from the whole current interval
+_INTERVAL_LENGTH = .05
+
+
+def get_supported_decay_params():
+    """Provides all names of currently supported parameters to specify
+    the relationship to determine the smoothing parameter at Exponential
+    Moving Average.
+
+    :returns list of str: the names of all supported parameters
+    """
+    return list(_DECAY_PARAMS_INFO.keys())
+
+
+def window_width_change(window_width, r_square):
```

Comment:
Fix: get_window_width_change or compute_window_width_change would be a little cleaner.